supabase/migrations/20240912225853_issue_comments.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE vectordump
+RENAME TO issue_comments;
+
+CREATE TABLE IF NOT EXISTS issues (
+    id VARCHAR primary key,
+    plaintext text,
+    embedding Vector(1024) not null,
+    payload jsonb,
+    author_id VARCHAR not null,
+    type text not null default 'issue',
```

Comment:
Why is the default issue on the comments table? Why even add the type column if it's in the comments table?

Reply:
Its there to ignore, bot comments, and chore issues created. So, while doing vector search we can ignore such issues and comments. I can remove that, but its a feature that could be useful while performing vector search.